src/test/resources/unrc/dose/Escenario.feature

```diff
@@ -0,0 +1,38 @@
+#Authors: Borda, Celi, de Prada, Goldenberg.
```

Comment:
Please change the name of the file to include the surname of at least one author